### PR TITLE
i18n: Eliminate duplicate or incorrect texts

### PR DESCRIPTION
**Pull request by greezybacon**
This is my attempt at removing similar texts from the translator work load. It also makes an attempt at dropping some misspelled words and works toward standardizing some phrases.

### CODE REVIEW

File: scp/roles.php [hunk 1]
```diff
@@ -39,11 +39,11 @@
             $msg = __('Role updated successfully');
         } elseif ($errors) {
             $errors['err'] = $errors['err'] ?:
-                sprintf(__('Unable to update %s. Correct error(s) below and try again!'),
+                sprintf(__('Unable to update %s. Correct any errors below and try again.'),
```
Comment (Chefkeks) [inline scp/roles.php]: Sometimes its "Correct ANY errors..." and sometimes its "Correct THE errors..." (e.g. /scp/tasks @ Line103). I guess it'd make sense to just use one phrase / sentence and not several to hint at errors which shall be corrected ;)
Rest of this PR looks pretty good so far :+1:

**Comment by greezybacon** [comment]
@Chefkeks thanks for the heads up. I took your lead

File: include/staff/slaplans.inc.php [hunk 2]
```diff
@@ -36,7 +36,7 @@
 
 $pageNav = new Pagenate($count, $page, PAGE_LIMIT);
 $pageNav->setURL('slas.php', $qs);
-$showing = $pageNav->showing().' '._N('SLA plan', 'SLA plans', $count);
+$showing = $pageNav->showing().' '._N('Service Level Agreement', 'Service Level Agreements', $count);
```
Comment (Chefkeks) [inline include/staff/slaplans.inc.php]: You changed "SLA Plan" / "SLA Plans" to the full length "Service Level Agreement" / "Service Level Agreements" here and in include/class.slas.php, but not in scp/slas.php. Is this for a specific reason or not? If not, you should may change it in scp/slas.php too ;)

Comment (greezybacon) [inline include/staff/slaplans.inc.php]: Personally, I think Service Level Agreement is more correct; however, honestly, after about six hours, I just got worm out of rephrasing text

**Pull request merged**